src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_evp.h

```diff
@@ -50,6 +50,22 @@ Direct shim to EVP_DigestFinal_ex.
 */
 PALEXPORT int32_t CryptoNative_EvpDigestFinalEx(EVP_MD_CTX* ctx, uint8_t* md, uint32_t* s);
 
+/*
+Function:
+EvpDup
+
+Returns an EVP_MD_CTX* representing the same state as ctx.
+*/
+PALEXPORT EVP_MD_CTX* CryptoNative_EvpDup(const EVP_MD_CTX* ctx);
```

Comment:
Where is this consumed by C#?

Reply:
It was part of the initial approach (look just like Windows), then I went the route of making an approach with fewer managed to native transitions.

I had left it as potentially useful later, but the only duplication reason I can really think of is this one, so I'll just mark the function static and remove the export.